src/SmartComponents/AdvisorySystems/AdvisorySystem.test.js

```diff
@@ -8,9 +7,12 @@ import { initMocks } from '../../Utilities/unitTestingUtilities.js';
 import AdvisorySystems from './AdvisorySystems';
 import { BrowserRouter as Router } from 'react-router-dom';
 import NoRegisteredSystems from '../../PresentationalComponents/Snippets/NoRegisteredSystems';
+import ShallowRenderer from 'react-test-renderer/shallow';
```

Comment:
Why does this test need to use `react-test-renderer`?

Reply:
You are correct, asFragment is better in this case, catches more details in the snapshots